assets/src/scripts/charcoal/admin/widget/form.js

```diff
@@ -435,6 +436,10 @@ Charcoal.Admin.Widget_Form.prototype.request_success = function ($form, $trigger
                 '&obj_id=' + response.obj_id;
         }
     } else {
+        if (this.force_refresh) {
+            window.location.href = window.location.href;
+        }
+
```

Comment:
If the response has feedback (`response.feedbacks`), the force refresh should occur after a confirmation alert, similar to `response.next_url` above.

```suggestion
        if (this.force_refresh) {
			if (response.feedbacks) {
				// @todo "dynamise" the label
				Charcoal.Admin.feedback().add_action({
					label:    commonL10n.continue,
					callback: function () {
						window.location.reload();
					}
				});
			} else {
                window.location.reload();
            }
        }

```

Also, `window.location.reload()` seems like the better option for reloading the page.